telcell/cell_identity.py

```diff
@@ -57,16 +57,16 @@ def create(*,
             return NRCell(mcc, mnc, eci)
         elif radio is not None:
             raise ValueError(f"unsupported radio technology: {radio}")
-        elif eci is not None and (ci is None or lac is None):
-            # we have `eci` but not `lac`, `ci` --> guess LTE/NR
+        elif eci is not None:  # we have: `eci`; missing: `radio`
+            if ci is not None or lac is not None:
+                raise ValueError("either `ci` or `eci` should be provided, but not both")
```

Comment:
The second part of this check is unclear to me: `lac` not being `None`  is not mentioned in the error message?